pkg/xray/resource_xray_security_policy_test.go

```diff
@@ -381,6 +319,43 @@ resource "xray_security_policy" "test" {
 `, name, description, ruleName, allowedLicense)
 }
 
+func verifySecurityPolicy(fqrn string, tempStruct map[string]string, cvssOrSeverity string) resource.TestCheckFunc {
+	if cvssOrSeverity == "cvss" {
+		return resource.ComposeTestCheckFunc(
```

Comment:
`ComposeTestCheckFunc` accepts a slice of `TestCheckFunc` (https://github.com/hashicorp/terraform-plugin-sdk/blob/main/helper/resource/testing.go#L646) so instead of having 2 almost identical list of `resource.TestCheckResourceAttr()`, you can create a common slice first. Then add the additional ones for "cvss" or "severity".

Reply:
yeah, implemented, thanks